platform.md

```diff
@@ -933,9 +928,8 @@ optional = false
 
 [[processes]]
 type = "<process type>"
-command = "<command>"
+command = ["<command>"]
```

Comment:
I think we need to describe (maybe in the builder section) the [conversions](https://github.com/buildpacks/rfcs/blob/main/text/0093-remove-shell-processes.md#api-compat) that are needed from `launch.toml` -> `metadata.toml` to provide a consistent user interface and avoid breaking older buildpacks.

Reply:
Added the description in the builder section.